<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefInputDiag</class>
 <widget class="QDialog" name="RefInputDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLineEdit" name="txtPos2"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtPos1"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="txtAtt2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Attitude</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtAtt1"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="txtPos3"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="txtAtt3"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtPos1</tabstop>
  <tabstop>txtPos2</tabstop>
  <tabstop>txtPos3</tabstop>
  <tabstop>txtAtt1</tabstop>
  <tabstop>txtAtt2</tabstop>
  <tabstop>txtAtt3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RefInputDiag</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RefInputDiag</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
